src/main/proto/ga4gh/variants.proto

```diff
@@ -191,4 +191,29 @@ message Variant {
   // determination of genotype with respect to this variant. `Call`s in this
   // array are implicitly associated with this `Variant`.
   repeated Call calls = 12;
+
+  // The "variant_type" is used to denote e.g. structural variants.
+  // Examples:
+  //   DUP  : duplication of sequence following "start"; not necessarily in situ
+  //   DEL  : deletion of sequence following "start"
+  string variant_type = 13;
```

Comment:
@david4096 Thought about that, but also the "baroque" :-)

I am open to any solution, as long as it is accepted & forward looking, and gets implemented. The implementation of other variant types is extremely important - SNVs are just a small subset (regarding genome coverage, functional impact; don't get blinded by their count number...).